docs/Agent/user-guide/man_page.md

```diff
@@ -1 +1,472 @@
-# Man Page
+# Man pages
+ 
+## Commands by functional group
+
+### Performance tool management commands
+-  [pbench-clear-results](#pbench-clear-results)
+-  [pbench-clear-tools](#pbench-clear-tools)
+-  [pbench-list-tools](#pbench-list-tools)
+-  [pbench-list-triggers](#pbench-list-triggers)
+-  [pbench-register-tool](#pbench-register-tool)
+-  [pbench-register-tool-set](#pbench-register-tool-set)
+-  [pbench-register-tool-trigger](#pbench-register-tool-trigger)
+### Benchmark commands
+-  [pbench-user-benchmark](#pbench-user-benchmark)
+### Upload to Pbench Server
+#### Pbench Server 0.69
+- [pbench-move-result](#pbench-move-result)
+- [pbench-copy-results](#pbench-copy-results) 
+#### Pbench Server 1.0
+-  [pbench-results-move](pbench-results-move)
+
+## Commands
+
+---
+
+### pbench-clear-results
+
+---
+
+**NAME**  
+`pbench-clear-results`, Clears the result directory
+
+**SYNOPSIS**  
+`pbench-clear-results [OPTIONS]`
+
+**DESCRIPTION**  
+This command clears the results directory from `/var/lib/pbench-agent` directory
+
+**OPTIONS**  
+ `-C`, `--config` PATH  
+Path to the Pbench Agent configuration file.
+This option is required if not provided by the `_PBENCH_AGENT_CONFIG` environment variable.
+
+
+`--help`  
+Show this message and exit.
+
+---
+
+### pbench-clear-tools
+
+---
+
+**NAME**  
+`pbench-clear-tools`, Clear registered tools by name or group
+
+**SYNOPSIS**  
+`pbench-clear-tools [OPTIONS]`
+
+**DESCRIPTION**  
+Clear all tools which are registered and can filter by name of the group
+
+**OPTIONS**  
+`-C`, `--config` PATH  
+Path to the Pbench Agent configuration file.
+This option is required if not provided by the `_PBENCH_AGENT_CONFIG` environment variable.
+
+`-n`, `--name`, `--names` STR  
+Clear only the <`STR`> tool.
+
+`-g`, `--group`, `--groups` STR  
+Clear the tools in the <`STR`> group. If no group is specified, the `default` group is assumed.
+
+`-r`, `--remote`, `--remotes` STR  
+Clear the tool(s) only on the specified remote(s). Multiple remotes may be specified as a comma-separated list. If no remote is specified, all remotes are cleared
+
+`--help`  
+Show this message and exit.
+
+---
+
+### pbench-copy-results
+
+---
+
+**NAME**  
+`pbench-copy-results`, Copies result tarball to the 0.69 Pbench Server.
+
+**SYNOPSIS**  
+`pbench-copy-results [--help] --user=<user> [--controller=<controller>] [--prefix=<path>][--xz-single-threaded] [--show-server]`
+
+**DESCRIPTION**  
+Push the benchmark result to the Pbench Server without removing it from the local host. This command requires `/opt/pbench-agent/id_rsa` file with the private SSH key,when pushing to a 0.69 Pbench Server.
+
+**OPTIONS**  
+`--user`  
+This option value is required if not provided by the
+`PBENCH_USER` environment variable; otherwise, a value provided
+on the command line will override any value provided by the
+environment.
```

Comment:
This is true, but the description is a bit messy if you don't understand the unpacked directory tree on the 0.69 server. Maybe "when referencing unpacked results from a server, you can use this value to view results attributed to a specific user. For example, with a result name `<result>` with a controller of `<controller>` and `--user <user>`, you can access results either as `http://<server>/results/<controller>/<result>` or as `http://<server>/users/<user>/<controller>/<result>`."

And `--controller` has a similar description, but changes the value of `<controller>`, instead of using the default `hostname` which in a cloud environment often is not particularly useful or unique.

`--prefix` is even weirder, as it's a directory sub-path that's inserted *between* the controller and the unpacked results, as `http://<server>/results/<controller>/<prefix>/<result>` ... and it can be any arbitrary directory path, like `results/<controller>/this/is/a/few/extra/levels/of/directory/<result>`.

This is a widely known and used feature of the 0.69 server architecture to help categorize data, and one we decided to drop for 1.0 in favor of metadata. I wouldn't object to filling in "advanced" details like that after we've got the basic framework in place, however.